app/src/main/java/io/apicurio/registry/metrics/health/liveness/PersistenceExceptionLivenessCheck.java

```diff
@@ -31,6 +32,7 @@ public class PersistenceExceptionLivenessCheck extends AbstractErrorCounterHealt
      * before the liveness check fails.
      */
     @ConfigProperty(name = "registry.metrics.PersistenceExceptionLivenessCheck.errorThreshold", defaultValue = "1")
+    @Info( category = "health", description = "Error Threshold of persistence liveness check", availableSince = "1.0.2.Final")
     Integer configErrorThreshold;
```

Comment:
Suggest "Error threshold..." (occurs in a few other descriptions)

Reply:
👍